docs/Troubleshooting.md

```diff
@@ -1,137 +1,140 @@
-   In this case, profiling any Java process which had started prior to the installation
-   will continue to display this message, since the process had loaded
-   the older version of the JDK which lacked debug symbols.
-   Restarting the affected Java processes should resolve the issue.
-
-*  ```
-   VMStructs unavailable. Unsupported JVM?
-   ```
-   JVM shared library does not export `gHotSpotVMStructs*` symbols -
-   apparently this is not a HotSpot JVM. Sometimes the same message
-   can be also caused by an incorrectly built JDK
-   (see [#218](https://github.com/async-profiler/async-profiler/issues/218)).
-   In these cases installing JDK debug symbols may solve the problem.
-
-*  ```
-   Could not parse symbols from <libname.so>
-   ```
-   Async-profiler was unable to parse non-Java function names because of
-   the corrupted contents in `/proc/[pid]/maps`. The problem is known to
-   occur in a container when running Ubuntu with Linux kernel 5.x.
-   This is the OS bug, see https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1843018.
-
-*  ```
-   Could not open output file
-   ```
-   Output file is written by the target JVM process, not by the profiler script.
-   Make sure the path specified in `-f` option is correct and is accessible by the JVM.
-
-
-* No Java stacks will be collected if `-XX:MaxJavaStackTraceDepth` is zero
-or negative. The exception is `--cstack vm` mode, which does not take
-`MaxJavaStackTraceDepth` into account.
-
-
-* Too short profiling interval may cause continuous interruption of heavy
-system calls like `clone()`, so that it will never complete;
-see [#97](https://github.com/async-profiler/async-profiler/issues/97).
-The workaround is simply to increase the interval.
-
-
-*  When agent is not loaded at JVM startup (by using -agentpath option) it is
-highly recommended to use `-XX:+UnlockDiagnosticVMOptions -XX:+DebugNonSafepoints` JVM flags.
-Without those flags the profiler will still work correctly but results might be
-less accurate. For example, without `-XX:+DebugNonSafepoints` there is a high chance
-that simple inlined methods will not appear in the profile. When the agent is attached at runtime,
-`CompiledMethodLoad` JVMTI event enables debug info, but only for methods compiled after attaching.
-
-
-* On most Linux systems, `perf_events` captures call stacks with a maximum depth
-   of 127 frames. On recent Linux kernels, this can be configured using
-   `sysctl kernel.perf_event_max_stack` or by writing to the
-   `/proc/sys/kernel/perf_event_max_stack` file.
-
-
-* You will not see the non-Java frames _preceding_ the Java frames on the
-stack, unless `--cstack vm` is specified.
-For example, if `start_thread` called `JavaMain` and then your Java
-code started running, you will not see the first two frames in the resulting
-stack. On the other hand, you _will_ see non-Java frames (user and kernel)
-invoked by your Java code.
-
-
-* macOS profiling is limited to user space code only.
+- ```
```

Comment:
Having one style is good, but I must admit that in some cases lists with `*` look better.
For instance, here `-` is blending with backticks.

Reply:
I can try to make `*` the default list token.

Reply:
Used idiomatic headers here, instead of code blocks in list items.